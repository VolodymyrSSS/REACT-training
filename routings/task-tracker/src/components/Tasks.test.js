import React from 'react';
import renderer from 'react-test-renderer'; // ES6
import Tasks from './Tasks';

it('should match its snapshot with tasks', () => {
  const tasks = [{
    text: 'some test-text One',
    day: 'March 29th, at 10.15am',
    reminder: false
  },{
    text: 'some test-text Two',
    day: 'March 30th, at 17.20pm',
    reminder: true
  }];
  
  const tree = renderer.create(
    <Tasks tasks={tasks} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
