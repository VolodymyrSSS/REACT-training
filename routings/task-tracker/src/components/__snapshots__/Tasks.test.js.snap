// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match its snapshot with tasks 1`] = `
Array [
  <div
    className="task "
    onDoubleClick={[Function]}
  >
    <h3>
      some test-text One
      <svg
        fill="currentColor"
        height="1em"
        onClick={[Function]}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": "red",
          }
        }
        viewBox="0 0 352 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
        />
      </svg>
    </h3>
    <p
      style={
        Object {
          "color": "blue",
        }
      }
    >
      March 29th, at 10.15am
    </p>
  </div>,
  <div
    className="task reminder"
    onDoubleClick={[Function]}
  >
    <h3>
      some test-text Two
      <svg
        fill="currentColor"
        height="1em"
        onClick={[Function]}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": "red",
          }
        }
        viewBox="0 0 352 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"
        />
      </svg>
    </h3>
    <p
      style={
        Object {
          "color": "blue",
        }
      }
    >
      March 30th, at 17.20pm
    </p>
  </div>,
]
`;
